requirements.txt

```diff
@@ -1,4 +1,7 @@
+mkl
+blas=*=mkl
 numpy
+scipy
```

Comment:
Scipy is only used for a few algorithms and it is not present in some clusters,
So we don't want to put it as a requirement.
However, since we want to use it for the regtests we have to install it in as it was done before.

Please take 'scipy' out from the "requirements.txt" and install it in pip

Reply:
done so... removed scipy from requirements.txt